st3215_control:
  ros__parameters:
    speed: 4000
    acceleration: 2000
    usb_port: /dev/ttyACM0
    baud_rate: 1000000
    joint_names: ["Rotation", "Pitch", "Elbow", "Wrist_Pitch", "Wrist_Roll", "Jaw"]
    joint_ids: [3, 6, 9, 18, 15, 1]
    invert_motor: [true, false, false, false, false, false]